Dockerfile

```diff
@@ -6,7 +6,6 @@ RUN apk add --no-cache \
       zip \
       ttf-freefont \
       ttf-opensans \
-      ttf-ubuntu-font-family \
```

Comment:
Why are you removing this font?

Reply:
it was removed from the alpine repository

Reply:
ok, I also don't see a reason why it was added :)